### PR TITLE
Am example tate config

### PR DESCRIPTION
**Pull request by austin-millan**

### CODE REVIEW

**Comment by austin-millan** [comment]
This is a duplicate of #70

File: tate_config.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
+{
+	"TemplateData": {
```
Comment (benjspriggs) [inline tate_config.json]: Isn't this `template` instead of `TemplateData`?

File: tate_config.json [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+			"translations": {
+				"RTSTART": "specjbb.controller.rtcurve.start",
+				"T1": "specjbb.forkjoin.workers.Tier1",
+				"T2": "specjbb.forkjoin.workers.Tier2",
+				"T3": "specjbb.forkjoin.workers.Tier3",
+				"NUMA Nodes": "specjbb.group.count"
+			}
+		}
+	},
+	"RunList": [{
```
Comment (benjspriggs) [inline tate_config.json]: Isn't this `RunList` instead of `runs`?

**Comment by austin-millan** [comment]
This template is based on the [wiki page](https://github.com/PDXCapstoneF/SPECtate/wiki/Tate-Config#tateconfig). Want me to push an update with these key changes?

**Comment by benjspriggs** [comment]
@austin-millan You're right, I had misread the wiki.

**Pull request merged**